### PR TITLE
Add no access token validation

### PR DESCRIPTION
**Pull request by manuelbaez**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ const { hideBin } = require('yargs/helpers')
 const { _: username, token } = yargs(hideBin(process.argv)).argv
 
 const fetchContributions = require('./fetch-contributions')
+
+if (!token) {
+  console.error(chalk.red('No access was token provided.'))
```
Comment (josbelluna) [inline index.js:12]: Explain why token is required

**Review by josbelluna — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (manuelbaez) [reply index.js]: Done

File: index.js [hunk 2]
```diff
@@ -7,6 +7,15 @@ const { hideBin } = require('yargs/helpers')
 const { _: username, token } = yargs(hideBin(process.argv)).argv
 
 const fetchContributions = require('./fetch-contributions')
+
+if (!token || typeof token !== 'string') {
+  console.error(
+    chalk.red(`
+      You need to provide a github access token to access the contributor's information, please add --token your-access-token.
```
Comment (josbelluna) [inline index.js:14]: please use YOUR_ACCESS_TOKEN, and change word access

Comment (josbelluna) [inline index.js:14]:
```suggestion
      You need to provide a github access token to view the contributor's information, please add --token YOUR_ACCESS_TOKEN.
```

**Review by josbelluna — CHANGES_REQUESTED** [review]

**Review by josbelluna — APPROVED** [review]
Looks good.

**Pull request merged**